### ISSUE TITLE
Accounts Receivable >> Sales Quotation

### ISSUE THREAD

**Issue by medhatelmasry** [medhatelmasry/GoodBooks#20]
Change Accounts Receivable >> Sales Quotation to Sales Proposal 
Ensure that the form contains these columns:

Customer 
Description
Quantities
Discounts % or $
Delivery Date
Price per unit
Amount excluding taxes

**Comment by ydongchoi** [comment]
**Database Diagram for SalesQuote - It should be replaced to SalesProposal**

![image](https://github.com/user-attachments/assets/f74c748c-06e0-4de1-90ce-8fa3521f9d44)

**Comment by ydongchoi** [comment]
**[Prototype of Sales Proposals]**

![image](https://github.com/user-attachments/assets/acd5d32e-0bbc-41fb-a7fa-ec84d95e7cdb)

**Comment by ydongchoi** [comment]
An expiry date column is needed in the sales proposal table.

**Comment by medhatelmasry** [comment]
Yes. I agree that an expiry date is needed with sales proposals.

Regards,
Medhat Elmasry
